src/ICeWall/doc/set_die_area.md

```diff
@@ -0,0 +1,11 @@
+## ICeWall set_die_area
+### Synopsys
+```
+  % ICeWall set_die_area <minX> <minY> <maxX> <maxY>
```

Comment:
Why not take it from the db?

Reply:
Same comment as for die_area - probably should be expressed as more of a check than a set

Reply:
Check makes sense to me.